pkg/linguist/dev.go

```diff
@@ -106,7 +159,10 @@ func normalizeLanguage(language string) string {
 	case "python":
 		return python
 	case "java":
```

Comment:
I think the linguist library already returns gradle and/or maven, can you try it out?